configs/exokit.json

```diff
@@ -1,8 +1,7 @@
 {
   "index_name": "exokit",
   "start_urls": [
-    "https://docs.webmr.io/docs/",
-    "https://docs.webmr.io/docs/5MinuteGuide.html"
+    "https://docs.webmr.io/docs/whatisexo.html"
```

Comment:
You will still need to include https://docs.webmr.io/docs/ since it defines the root of the the allowed URLs. I will do it for you

Reply:
OK, thanks for that.